projects/cm_mcu/LocalTasks.c

```diff
@@ -243,11 +249,15 @@ struct MonitorI2CTaskArgs_t ffl12_f1_args = {
     .n_commands = NCOMMANDS_FFL12_F1,
     .n_values = NSUPPLIES_FFL12_F1 * NPAGES_FFL12_F1 * NCOMMANDS_FFL12_F1,
     .n_pages = NPAGES_FFL12_F1,
+    .selpage_reg = FF_SELPAGE_REG,
     .sm_values = ffl12_f1_values,
     .smbus = &g_sMaster4,
     .smbus_status = &eStatus4,
     .xSem = NULL,
     .ffpart_bit_mask = 0U,
+    .n_rxchs = FF_12_NRXCH,
+    .opt_pow_values = ffl12_f1_optpow_values,
```

Comment:
why are these here and not just part of the regular data stored in .sm_values?

Reply:
From my recollection, I thought that we would send 4 values per for each 4-ch XCVR FFs and 12 values for each 12-ch Rx FFs so I separate this out from .sm_values that has a one-to-one device to register output. At this point, we store an average value from 4-ch/12-ch FFs. If you suggest removing this .opt_pow_value and add its values to .sm_values, I can do that too. 

However, for future development (in case the cm addresses in Zynq are increased), we can have this separate attribute  come into handy?

Reply:
we are starting to treat this struct more and more like a grab-bag with slots of entries that only count for one device or another, and the code is full of exceptions which will make it hard to debug and extend.

Reply:
Okay, I get your point. I will add optical power values back to .sm_values and remove this attribute.